src/add/operator/zip-static.ts

```diff
@@ -1,5 +1,5 @@
 import {Observable} from '../../Observable';
 import {zip} from '../../operator/zip-static';
-Observable.zip = zip;
+Observable.zip = <any>zip;
```

Comment:
While working on #1030, I remembered that this patch touches in some auto-generated files. I think you should remove this kind of change from this patch. You'll need to drop changes in any file inside `src/add/*` (I think you modified only zip and zip-static in this patch).

What do you think, @kwonoj?